### ISSUE TITLE
RC test: Graphing Slope-Intercept 1.0.0-rc.1

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/tasks#815]
RC version:
http://www.colorado.edu/physics/phet/dev/html/graphing-slope-intercept/1.0.0-rc.1/graphing-slope-intercept_en.html 

iFrame test:
http://www.colorado.edu/physics/phet/dev/html/graphing-slope-intercept/1.0.0-rc.1/graphing-slope-intercept_en-iframe.html

Text matrix:
https://docs.google.com/spreadsheets/d/1Tocnn1DCtML3rrYDJj4pTgQ96tE0caTQR4fqYeA0vuU/edit#gid=2

In addition to the test matrix, please perform these specific tests on 1 or 2 platforms:

- [x] ?stringTest=double (double all strings)
- [x] ?stringTest=long (excessively long strings)
- [x] ?stringTest=X (short strings)
- [x] ?stringTest=rtl (right-to-left text direction)
- [x] ?stringTest=xss (test for security issues, passes if it does NOT redirect to another page)
- [ ] full screen test
- [ ] screenshot test

Game challenges, and the rewards for a perfect score in each level, are specified in:
https://docs.google.com/document/d/1ztSRN1YE1BIGI1rsCBnZXzZQYvelSFySp4sDHPh2OJs/edit#

Sim-specific query parameters are in [GLQueryParameters.js](https://github.com/phetsims/graphing-lines/blob/master/js/common/GLQueryParameters.js).  These have been simplified a bit since 1.0.0-dev.1, so review them again.

Terminology is defined in [implementation-notes.md](https://github.com/phetsims/graphing-lines/blob/master/doc/implementation-notes.md).

For a list of the issues that were addressed during dev testing, see https://github.com/phetsims/tasks/issues/812.

Please mention this issue in any GitHub issues that you create.

**Comment by pixelzoom** [comment]
@ariel-phet please set priority and assign.

**Comment by pixelzoom** [comment]
@amanda-phet Graphing Slope-Intercept is now in RC testing.

**Comment by ariel-phet** [comment]
@phet-steele marking as high, currently after #807 is finished in the QA line

**Comment by pixelzoom** [comment]
Note to self: Changes in #11 will required changes to graphing-lines. So there will need to be at least one more RC "spot test", which uses the master branch of graphing-lines.

**Comment by pixelzoom** [comment]
@phet-steele It looks like RC testing has not started, so please hold off until I give the green light. https://github.com/phetsims/graphing-slope-intercept/issues/11 required nontrivial changes to both graphing-slope-intercept and graphing-lines, so I'm going to delete 1.0.0-rc.1 and rebuild it when that issue is resolved.

**Comment by pixelzoom** [comment]
@phet-steele I rebuilt 1.0.0-rc.1, so you may proceed with RC testing.

Note to self: Here's how I rebuilt 1.0.0-rc.1.  I'm using the same shas, but picking up the changes to graphing-lines from master.

```
% cd graphing-slope-intercept
% git checkout 1.0
% grunt checkout-shas
% cd ../graphing-lines
% git checkout master
% cd ../graphing-slope-intercept
% grunt
% grunt deploy-rc
```

**Comment by phet-steele** [comment]
@pixelzoom, as a reminder before rc.2, this sim will need to pick up changes from phetsims/vibe/issues/30 (of which rc.1 currently suffers from).

**Comment by pixelzoom** [comment]
Thanks. I was unaware of phetsims/vibe#30.

**Comment by phet-steele** [comment]
Actually, it may need the other two patches from #818?

**Comment by pixelzoom** [comment]
@jonathanolson Advice on how to integrate the fix for phetsims/vibe#30 into the release branch for this sim?

**Comment by pixelzoom** [comment]
@phet-steele Picking up the audio fix is not going to be straightforward, so I moved it to its own issue. See https://github.com/phetsims/graphing-slope-intercept/issues/17.

**Comment by pixelzoom** [comment]
Due to significant code changes required to address https://github.com/phetsims/graphing-lines/issues/78 (memory leak in Graphing Lines), @phet-steele and I agreed that this would be a good place to stop this RC test task.  And prior to starting the next RC test, we should do another dev test of Graphing Lines.

Leaving this issue assigned to me, so that I can identify what needs to be regression tested in the next RC test.

**Comment by pixelzoom** [comment]
The next RC test is deferred until we complete dev testing of Graphing Lines, see https://github.com/phetsims/tasks/issues/825.

**Comment by pixelzoom** [comment]
Graphing Lines 1.2.0 has been published, so we can proceed with the next RC task.
